packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -10,12 +10,16 @@ import 'package:vm_service/vm_service.dart';
 import 'profiler/cpu_profile_model.dart';
 import 'version.dart';
 
+typedef TrackFuture = Future<T> Function<T>(String, Future<T>);
+
 class VmServiceWrapper implements VmService {
   VmServiceWrapper(
     this._vmService,
     this.connectedUri, {
     this.trackFutures = false,
-  });
+  }) {
+    trackFuture = _trackFuture;
```

Comment:
why is this necessary? can't we just make the _trackFuture method public and add a visibleForTesting annotation?

Reply:
I wasn't sure if we'd want to make `_trackFuture` public, but I guess this effectively does that anyway. Removed and made `_trackFuture` public with `@visibleForTesting`.